fjord/base/middleware.py

```diff
@@ -163,4 +163,5 @@ def process_request(self, request):
 
         request.path_info = '/' + prefixer.shortened_path
         request.locale = prefixer.locale
-        tower.activate(prefixer.locale)
+        translation.activate(prefixer.locale)
+        install_gettext()
```

Comment:
Any idea why we need this line or why we have to do this every time we process a request?